src/main/java/at/hannibal2/skyhanni/features/garden/visitor/GardenVisitorDropStatistics.kt

```diff
@@ -233,6 +234,32 @@ object GardenVisitorDropStatistics {
         display = formatDisplay(drawDisplay(storage))
     }
 
+    fun reset(args: Array<String>) {
+        if (args.size == 1 && args[0].lowercase() == "confirm") {
+            val storage = GardenAPI.storage?.visitorDrops ?: return
+            acceptedVisitors = 0
+            deniedVisitors = 0
+            totalVisitors = 0
+            coinsSpent = 0
+            storage.copper = 0
+            storage.bits = 0
+            storage.farmingExp = 0
+            storage.gardenExp = 0
+            storage.gemstonePowder = 0
+            storage.mithrilPowder = 0
+            storage.visitorRarities = arrayListOf(0, 0, 0, 0, 0)
+            storage.rewardsCount = mapOf<VisitorReward, Int>()
+            ChatUtils.chat("Visitor Drop Statistics reset!")
+            saveAndUpdate()
+            return
+        }
+
+        ChatUtils.clickableChat(
```

Comment:
maybe wait until https://github.com/hannibal002/SkyHanni/pull/1128 is merged. then you can send a clickable chat message that automatic resets the data on click, without running a "conffirm" command in betwee.

maybe even extend this featur to only work for some time.

Reply:
@superhize 
https://github.com/hannibal002/SkyHanni/pull/1128 is merged